pajbot/web/__init__.py

```diff
@@ -147,13 +164,13 @@ def init(args):
     default_variables = {
         "version": version,
         "last_commit": last_commit,
-        "bot": {"name": config["main"]["nickname"]},
+        "bot": bot_login,
```

Comment:
This is not the same, `default_variables["bot"]` used to be a dictionary with the key `name` with the bot's login name. Now `default_variables["bot"]` is  the bot's login name.